src/controls/IRoomControllerV2.ts

```diff
@@ -0,0 +1,380 @@
+        this.addStateChangeListener(uuid + '.tempTarget', (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+            // Target has been set manually so start an override at that value
+            // setManualTemperature is redundant in this context as simply setting that value
+            // does not actually tell IRC to switch to manual mode.
+            // We could force whatever updates the tempTarget to then need to change mode to manual
+            // but that would be illogical as the target now shown would not actually be the target
+            // being worked with.
+            // So because of this, just start an override at the given temperature.
+            this.sendCommand(control.uuidAction, 'override/3//' + newValue);
+        });
+
+        await this.updateStateObjectAsync(
+            uuid + '.operatingMode',
+            {
+                name: control.name + ': operatingMode',
+                read: true,
+                write: true,
+                type: 'number',
+                role: 'level',
+                states: {
+                    0: 'Automatic heating/cooling',
+                    1: 'Automatic heating',
+                    2: 'Automatic cooling',
+                    3: 'Manual heating/cooling',
+                    4: 'Manual heating',
+                    5: 'Manual cooling',
+                },
+            },
+            control.states.operatingMode,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+            },
+        );
+        this.addStateChangeListener(uuid + '.operatingMode', (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+            this.sendCommand(control.uuidAction, 'override/' + newValue);
+        });
+
+        await this.createButtonCommandStateObjectAsync(control.name, uuid, 'stopOverride');
+        this.addStateChangeListener(uuid + '.stopOverride', () => {
+            this.sendCommand(control.uuidAction, 'stopOverride');
+        });
+
+        // When in Eco/Building Protection modes Loxone app/web shows target as min..max range
+        // Handy so create derived state for each
+        await this.updateObjectAsync(uuid + '.tempTargetMin', {
+            type: 'state',
+            common: {
+                name: control.name + ': tempTargetMin',
+                type: 'number',
+                role: 'value.temperature',
+                read: true,
+                write: false,
+            },
+            native: {},
+        });
+        await this.updateObjectAsync(uuid + '.tempTargetMax', {
+            type: 'state',
+            common: {
+                name: control.name + ': tempTargetMax',
+                type: 'number',
+                role: 'value.temperature',
+                read: true,
+                write: false,
+            },
+            native: {},
+        });
+
+        // TODO: Controls not yet implemented:
+        // override (at least not fully)
```

Comment:
See comment above.
I would in any case not implement the timer stuff as it can be done in another way in ioBroker.

Reply:
So leaving this as a 'TODO' forever will be good for you then? ;)